Assets/Scripts/PopulationController.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using Database.Tables;
+using DataStructures;
+
+
+public class PopulationController : MonoBehaviour, IPointerClickHandler
+{    
+    public DatabaseManager _databaseManager;
+    private Canvas _canvas;
+    // Start is called before the first frame update
+    void Start()
+    {
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
+    private void Awake() {
+        _canvas = transform.Find("Canvas").GetComponent<Canvas>();
```

Comment:
Instead of running the `Find`, lets make the Canvas a game object in the scene and then add reference to the Canvas.

This can be done by adding the `[SerializeField]` attribute to the `private Canvas _canvas`.

The following image shows you how to add the reference using the inspector:
![image](https://user-images.githubusercontent.com/74933136/143115172-29133d13-0244-444e-85ad-608316ff9391.png)